great_expectations/render/renderer/site_builder.py

```diff
@@ -188,7 +200,20 @@ def build(self, resource_identifiers=None):
                             and avoids full rebuild.
         :return:
         """
-
+        
+        # copy static assets
+        project_root_dir = self.data_context.root_directory
+        site_base_dir = self.store_backend["base_directory"]
+        site_dir = os.path.join(project_root_dir, site_base_dir)
+        static_assets_source_dir = file_relative_path(__file__, "../view/static")
+        static_assets_destination_path = os.path.join(site_dir, "static")
+        if os.path.isdir(static_assets_destination_path):
+            message = """Warning. An existing static assets directory was found here: {}.
+                    - Static assets were not copied.""".format(static_assets_destination_path)
+            logger.warning(message)
+        else:
+            shutil.copytree(static_assets_source_dir, static_assets_destination_path)
+        
```

Comment:
Moved copying of static assets tree from ge init scaffolding to SiteBuilder build process

Reply:
Can we go over this in a call tomorrow? Since store_backend may not have a base_directory, I think site_base_dir still isn't being set correctly here.